.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
 *                         @element-hq/element-web-reviewers
+synapse/                  @element-hq/synapse-core
```

Comment:
This is causing an error as the synapse-core team does not have access to the repo, should it not be the SPD team?

Reply:
Thanks, fixed in 5f405b4